tests/pruners_tests/test_median.py

```diff
@@ -116,3 +116,33 @@ def test_median_pruner_n_warmup_steps():
     # A pruner is activated after warm-up steps.
     assert pruner.prune(
         study=study, trial=study._storage.get_trial(trial.trial_id))
+
+
+@pytest.mark.parametrize(
+    'n_warmup_steps,interval_steps,report_steps,expected_prune_steps', [
+        (1, 2, 1, (2, 4)),
+        (0, 3, 10, list(range(1, 30))),
+        (2, 3, 10, list(range(11, 30))),
+        (0, 10, 3, [1, 2, 3, 13, 14, 15, 22, 23, 24]),
+        (2, 10, 3, [4, 5, 6, 13, 14, 15, 25, 26, 27]),
+    ])
+def test_median_pruner_interval_steps(
+        n_warmup_steps, interval_steps, report_steps, expected_prune_steps):
+    # type: (int, int, int, Tuple[int, ...]) -> None
```

Comment:
```suggestion
    # type: (int, int, int, List[int]) -> None
```

Reply:
This and the above suggestion is based on https://github.com/pfnet/optuna/pull/660/files#r344002781.

Reply:
Thanks! I sent a separate commit since we needed additional imports from `typing`.